app/src/androidTest/java/org/openobservatory/ooniprobe/client/OONIAPIClientTest.java

```diff
@@ -66,4 +69,51 @@ public void onError(String msg) {
     @Test
     public void testDeleteJsons() {
     }
+
+    @Test
+    public void getMeasurementJsonSuccess() {
+        final CountDownLatch signal = new CountDownLatch(1);
+        a.getOkHttpClient().newCall(new Request.Builder().url(JSON_URL).build()).enqueue(new GetMeasurementJsonCallback() {
+            @Override
+            public void onSuccess(String json) {
+                Assert.assertNotNull(json);
+                signal.countDown();
+            }
+
+            @Override
+            public void onError(String msg) {
+                Assert.fail();
+                signal.countDown();
+            }
+        });
+        try {
+            signal.await(1, TimeUnit.MINUTES);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+            Assert.fail();
+        }
+    }
+
+    @Test
+    public void getMeasurementJsonError() {
+        final CountDownLatch signal = new CountDownLatch(1);
+        a.getOkHttpClient().newCall(new Request.Builder().url(NON_PARSABLE_URL).build()).enqueue(new GetMeasurementJsonCallback() {
```

Comment:
Can you rename `NON_PARSABLE_URL` like `NON_EXISTING_MEASUREMENT_URL` (or even something shorter but meaningful)? The name has now become misleading.